### PR TITLE
#259 - Add UserControllerTest::getCollectionAction()

### PR DESCRIPTION
**Pull request by crouillon**

### CODE REVIEW

**Comment by eric-chau** [comment]
Be used to provide a description please so we can all understand what happens.

File: Rest/Tests/Controller/UserControllerTest.php [hunk 1]
```diff
@@ -118,6 +118,23 @@ public function testGetAction()
...
+     */
+    public function testGetCollectionAction()
+    {
+        $controller = $this->getController();
+
+        // no filters
+        $response = $this->getBBApp()->getController()->handle(new Request(array(), array(
+        ), array(
+            '_action' => 'getCollectionAction',
+            '_controller' => 'BackBee\Rest\Controller\UserController',), array(), array(), array('REQUEST_URI' => '/rest/1/test/')));
```
Comment (mickaelandrieu) [inline Rest/Tests/Controller/UserControllerTest.php]: are you sure ?

**Comment by crouillon** [comment]
See with @ndufreche, we need to test the return type of UserController::getCollectionAction() to be sure to get an array and not an object

**Comment by eric-chau** [comment]
Ok thanks at you both :+1:

**Pull request merged**